app/handlers/errors.py

```diff
@@ -1,39 +1,44 @@
-from aiogram import types
-from aiogram.utils.exceptions import CantParseEntities, BadRequest
-from aiogram.utils.markdown import quote_html
+from functools import partial
 
-from app.config.main import load_config
-from app.misc import dp, bot
+from aiogram import Dispatcher, Bot
+from aiogram.exceptions import TelegramBadRequest
+from aiogram.types.error_event import ErrorEvent
+from aiogram.utils.text_decorations import html_decoration as hd
+
+from app.models.config import Config
+from app.utils.exceptions import Throttled
 from app.utils.log import Logger
 
 
 logger = Logger(__name__)
 
 
-@dp.errors_handler()
-async def errors_handler(update: types.Update, exception: Exception):
+async def errors_handler(error: ErrorEvent, bot: Bot, config: Config):
     try:
-        raise exception
-    except CantParseEntities as e:
-        logger.error("Cause exception {e} in update {update}", e=e, update=update)
-        return True
-    except BadRequest as e:
+        raise error.exception
+    except Throttled:
+        pass
+    except TelegramBadRequest as e:
         if "rights" in e.args[0] and "send" in e.args[0]:
-            if update.message and update.message.chat:
-                logger.info("bot are muted in chat {chat}", chat=update.message.chat.id)
+            if error.update.message and error.update.message.chat:
+                logger.info("bot are muted in chat {chat}", chat=error.update.message.chat.id)
             else:
-                logger.info("bot can't send message (no rights) in update {update}", update=update)
+                logger.info("bot can't send message (no rights) in update {update}", update=error.update)
             return True

```

Comment:
In aiogram 3.x we should remove this.
This was neccessary in 2.x.

Reply:
fixed, thanks